lib/take_elevated.lua

```diff
@@ -41,21 +46,35 @@ local function calculateNewBucketContent(current, tokens_per_ms, bucket_size, cu
     end
 end
 
+local function takeERLQuota(erlQuotaKey, erl_quota_amount, erl_quota_expiration_epoch, current_timestamp_ms)
+    local erlQuotaExists = redis.call('EXISTS', erlQuotaKey)
```

Comment:
redis returns a [null reply](https://redis.io/commands/get/) for non-existent keys when using the `get` command. Could we use only `get` and not `exists` to reduce redis calls?

Reply:
good catch. I'll fix that

Reply:
fixed